Test/scripts/phpcs_meqp2.sh

```diff
@@ -4,14 +4,34 @@ set -e
 set -u
 set -x
 
-sudo composer self-update -q
 composer show -i
 
-echo "{\"http-basic\":{\"repo.magento.com\":{\"username\":\"${MAGENTO_PUBLIC_KEY}\",\"password\":\"${MAGENTO_PRIVATE_KEY}\"}}}" > $HOME/.composer/auth.json
```

Comment:
Why are we removing this and the self update? Trying to understand this.

Reply:
We did not need it even before the change